templates/configmap-env.yaml

```diff
@@ -344,3 +344,22 @@ data:
   {{- if .Values.mastodon.deepl.enabled }}
   DEEPL_PLAN: {{ .Values.mastodon.deepl.plan }}
   {{- end }}
+
+  {{- if .Values.mastodon.hcaptcha.enabled }}
+  HCAPTCHA_SITE_KEY: {{ .Values.mastodon.hcaptcha.siteId }}
+  {{- end }}
+
+  {{- if .Values.mastodon.cacheBuster.enabled }}
+  CACHE_BUSTER_ENABLED: "true"
+  {{- else }}
+  CACHE_BUSTER_ENABLED: "false"
+  {{- end }}
+  {{- if .Values.mastodon.cacheBuster.httpMethod }}
+  CACHE_BUSTER_HTTP_METHOD: {{ .Values.mastodon.cacheBuster.httpMethod }}
+  {{- end }}
+  {{- if .Values.mastodon.cacheBuster.authHeader }}
+  CACHE_BUSTER_SECRET_HEADER: {{ .Values.mastodon.cacheBuster.authHeader }}
+  {{- end }}
+  {{- range $k, $v := .Values.mastodon.extraEnvVars }}
```

Comment:
You forgot to remove this above, so those lines are now duplicated.

We need them at the end of the file, probably with a new line before to separate them

Reply:
Oh good catch, that's my bad